bucket/audacity242.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
+    "license": "GPL-2.0-or-later",
+    "url": "https://archive.org/download/audacity-2.4.2_202205/audacity-2.4.2.zip",
+    "hash": "0c14f7c6850c93b9dacc14fe66876b8dc3397d92dbd849898783a21bad1fff55",
+    "bin": "audacity.exe",

```

Comment:
CLI args?

Reply:
> https://forum.audacityteam.org/viewtopic.php?t=77335
Audacity only has limited command-line support. The only current command-line functions are that you can open audio files or Audacity project files (including multiple files) by passing them as arguments, 
for example:
audacity file1.aiff file2.wav file3.ogg

> The following options are also supported:
-help
display a brief list of command line options
-version
display the Audacity version number
-test
run "Benchmark" diagnostics tests
-blocksize
set the Audacity blocksize for writing files to disk of nnn bytes